wasm/src/main/java/com/dylibso/chicory/wasm/Parser.java

```diff
@@ -863,9 +864,51 @@ private static Instruction parseInstruction(ByteBuffer buffer) {
         }
         var operandsArray = new long[operands.size()];
         for (var i = 0; i < operands.size(); i++) operandsArray[i] = operands.get(i);
+        verifyAlignement(op, operandsArray);
         return new Instruction(address, op, operandsArray);
     }
 
+    private static void verifyAlignement(OpCode op, long[] operands) {
```

Comment:
Note there is a typo here, `verifyAlignement` should be named `verifyAlignment`

Reply:
Ouch, thanks for noticing, I'll fix it when I'm back in front of a PC.